### DISCUSSION TITLE
Allow allow_embedding = true

### DISCUSSION THREAD

**Discussion by jiquem** [teslamate-org/teslamate#1714] [Ideas]
Hello,

Could you please allow graphana to be embedded (would be usefull to add a link in Home Assistant sidebar) by adding this configuration parameter ?

allow_embedding = true

Thanks

**Comment by adriankumpf** [comment]
You can set `allow_embedding` via an environment variable e.g. in the docker-compose.yaml:

```
GF_SECURITY_ALLOW_EMBEDDING=true
```
